networks/2_quick_look_at_modules.py

```diff
@@ -0,0 +1,39 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:core_acc_env] *
+#     language: python
+#     name: conda-env-core_acc_env-py
+# ---
+
+# ## Quick look at modules
+
+# %load_ext autoreload
+# %autoreload 2
+import os
+import pandas as pd
+
+# +
+corr_threshold_list = [0.9, 0.8, 0.7, 0.6, 0.5]
+
+for corr_threshold in corr_threshold_list:
+    print(f"Modules using correlation threshold: {corr_threshold}")
+    pao1_membership_filename = f"pao1_membership_{corr_threshold}.tsv"
+    pa14_membership_filename = f"pa14_membership_{corr_threshold}.tsv"
+
+    pao1_membership = pd.read_csv(
+        pao1_membership_filename, sep="\t", header=0, index_col=0
+    )
+    pa14_membership = pd.read_csv(
+        pa14_membership_filename, sep="\t", header=0, index_col=0
+    )
+
+    print(pao1_membership["module id"].value_counts())
```

Comment:
I think plotting these counts would probably give a better visual idea of what the size distribution looks like (particularly if you want to compare the distributions between different thresholds), but this is fine if you're just looking for something super quick.

Reply:
Thats a good suggestion. This was definitely me doing a super quick check, but adding that visualization would be helpful